platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -1743,6 +1742,7 @@ public boolean onScale(ScaleGestureDetector detector) {
             if (uiSettings.isScrollGesturesEnabled() && !mQuickZoom && trackingSettings.isLocationTrackingDisabled()) {
                 // around gesture
                 mNativeMapView.scaleBy(detector.getScaleFactor(), detector.getFocusX() / mScreenDensity, detector.getFocusY() / mScreenDensity);
+                trackGestureEvent(MapboxEvent.GESTURE_QUICK_ZOOM, detector.getFocusX(), detector.getFocusY());
```

Comment:
@bleege 
the if check above indicated !quickzoom and a place like this will flood the eventmanager. 
I'm guessing the correct place for this is were we are setting `mQuickZoom = false;` aka the end of the event.